src/utils/fetchClient.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Instead of disabling the 'no-explicit-any' rule, consider using a more specific type or generics for better type safety.